### ISSUE TITLE
 Chronograf (1.7.15) undefined is not an object (evaluating 'a.author.name')

### ISSUE THREAD

**Issue by wmwv** [influxdata/chronograf#5413]
What browser are you using (name and version)?
Safari 13.0.4

What operating system are you using?
Mac OS X 10.14.6

Please describe what you were trying to do when you encountered this error:

  Loaded 
https://squash.lsst.codes
  which redirected to
https://chronograf-demo.lsst.codes

```
https://chronograf-demo.lsst.codes/src.f605a2e1.js:2363:384
map@[native code]
l@https://chronograf-demo.lsst.codes/src.f605a2e1.js:2363:313
sa@https://chronograf-demo.lsst.codes/src.f605a2e1.js:717:55870
$a@https://chronograf-demo.lsst.codes/src.f605a2e1.js:717:68129
oo@https://chronograf-demo.lsst.codes/src.f605a2e1.js:717:91057
uo@https://chronograf-demo.lsst.codes/src.f605a2e1.js:717:91444
Yo@https://chronograf-demo.lsst.codes/src.f605a2e1.js:717:98526
$o@https://chronograf-demo.lsst.codes/src.f605a2e1.js:717:97906
jo@https://chronograf-demo.lsst.codes/src.f605a2e1.js:717:96927
ho@https://chronograf-demo.lsst.codes/src.f605a2e1.js:717:95740
enqueueSetState@https://chronograf-demo.lsst.codes/src.f605a2e1.js:717:47536
https://chronograf-demo.lsst.codes/src.f605a2e1.js:708:1711
https://chronograf-demo.lsst.codes/src.f605a2e1.js:802:5035
[native code]
h@https://chronograf-demo.lsst.codes/src.f605a2e1.js:761:1186
https://chronograf-demo.lsst.codes/src.f605a2e1.js:1140:156
https://chronograf-demo.lsst.codes/src.f605a2e1.js:1156:260
https://chronograf-demo.lsst.codes/src.f605a2e1.js:1138:1220
https://chronograf-demo.lsst.codes/src.f605a2e1.js:1164:1354
L@https://chronograf-demo.lsst.codes/src.f605a2e1.js:5:4405
https://chronograf-demo.lsst.codes/src.f605a2e1.js:5:4190
c@https://chronograf-demo.lsst.codes/src.f605a2e1.js:1164:363
promiseReactionJob@[native code]
```

**Comment by russorat** [comment]
@wmwv thanks for opening the issue. Could you upgrade to Chronograf 1.8 and see if you still have the problem? Looks like there might be an infinite redirect loop here, but I'd need more information to help troubleshoot.

**Comment by wmwv** [comment]
@russorat Thank you.  I will follow up with the team.

**Comment by wmwv** [comment]
@russorat 
The developer for this deployment is @afausti.  He reports that 1.8.0 shows the same behavior.
1.7.14 does not have this problem
Both 1.7.15 and 1.8.0 show this problem.

**Comment by afausti** [comment]
@russorat we confirm this is fixed after we upgraded to 1.8.4, also reported in #5468 and #5466.

**Comment by russorat** [comment]
Thanks!
